include/api.php

```diff
@@ -626,7 +626,7 @@ function api_get_user(App $a, $contact_id = Null, $type = "json"){
 		// count friends
 		$r = q("SELECT count(*) as `count` FROM `contact`
 				WHERE  `uid` = %d AND `rel` IN ( %d, %d )
-				AND `self`=0 AND NOT `blocked` AND `hidden`=0",
+				AND `self`=0 AND NOT `blocked` AND NOT `pending` AND `hidden`=0",
```

Comment:
Two conflicting handling of "boolean" values in one line. Maybe decide for one here?